jetty-io/src/main/java/org/eclipse/jetty/io/ManagedSelector.java

```diff
@@ -60,6 +61,20 @@
 public class ManagedSelector extends ContainerLifeCycle implements Dumpable
 {
     private static final Logger LOG = Log.getLogger(ManagedSelector.class);
+    private static final boolean FORCE_SELECT_NOW;
```

Comment:
Any reason this is a constant?
Would it be useful to be able to enable/disable this from JMX?

Reply:
While be able to change for debugging might be good.... I'd prefer the final to give the optimiser the best chance of eliding it for the 99.999% case.... think of the CO2 :)